R/R.sublime-syntax

```diff
@@ -145,10 +192,11 @@ contexts:
       push:
```

Comment:
It feels like now that comments are included via `prototype` you need to exclude them here, cause `$` only ever introduces special function or variable names.